Resources/Locale/en-US/components/mind-component.ftl

```diff
@@ -0,0 +1,6 @@
+﻿# MindComponent localization
+
+## Messages displayed when a body is examined and in a certain state
+comp-mind-examined-catatonic = They are totally catatonic. The stresses of life in deep-space must have been too much for them. Any recovery is unlikely.
```

Comment:
These messages are supposed to use the correct pronouns. It doesn't matter that much but it's something to keep in mind, *especially* if there's no way to handle that with the current way Fluent integration is handled - as that would mean there's an underlying problem.

Reply:
I asked about whether we had a pronoun fluent function yet, but we don't, so I'd add either a TODO here or implement the pronoun function.